Nimble/DSL+Wait.swift

```diff
@@ -38,13 +38,22 @@ internal class NMBWait: NSObject {
 /// Wait asynchronously until the done closure is called.
 ///
 /// This will advance the run loop.
-public func waitUntil(timeout timeout: NSTimeInterval, file: String = __FILE__, line: UInt = __LINE__, action: (() -> Void) -> Void) -> Void {
-    NMBWait.until(timeout: timeout, file: file, line: line, action: action)
+public func waitUntil(timeout timeout: NSTimeInterval, file: String = __FILE__, line: UInt = __LINE__, action: ((Any?...) -> Void) -> Void) -> Void {
```

Comment:
Yeah, it would be better to just replace with a default parameter (with non-optional `NSTimeInterval`). It existed because past swift / xcode versions needed the duplicated versions to provide more sane auto-complete options.

:+1: